sphinx/themes/basic/static/searchtools.js

```diff
@@ -48,7 +48,7 @@ if (!Scorer) {
 }
 
 const _removeChildren = (element) => {
-  while (element.lastChild) element.removeChild(element.lastChild);
+  while (element && element.lastChild) element.removeChild(element.lastChild);
```

Comment:
I could also have checked for null at https://github.com/sphinx-doc/sphinx/blob/3b01fbe2adf3077cad5c2cf345c6d000d429d7ac/sphinx/themes/basic/static/searchtools.js#L265-L265, but seems safer to do it here, so all calls to this function are protected

Reply:
EMCA2020 would allow 
```suggestion
  while (element?.lastChild) element.removeChild(element.lastChild);
```

CanIUse suggests 92.1% coverage vs 95.4% for EMCA2018 (`...` spread syntax) -- maybe worth it, maybe not. 

A